docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -98,25 +106,29 @@ and `pyroscope.scrape`.
     }
 
     ```
+
 3. Save the changes to the file.
-### Start Grafana Agent Flow
 
-1. Start a local Pyroscope instance for testing purposes
+### Start the collector
+
+1. Start a local Pyroscope instance for testing purposes:
     ```bash
     docker run -p 4040:4040 grafana/pyroscope
     ```
-2. Start Grafana Agent
-    ```bash
-    grafana-agent-flow run conifguration.river
-    ```
+
+2. Start the collector:
+  * To start Grafana Alloy, replace `configuration.alloy` with your configuration file name: <br> `alloy run configuration.alloy`
```

Comment:
Is this correct? 
```suggestion
  * To start Grafana Alloy, replace `configuration.alloy` with your configuration file name: <br> `alloy run --stability.level=public-preview configuration.alloy`
    For more information about `stability.level`, refer to [The run command](https://grafana.com/docs/alloy/latest/reference/cli/run/#permitted-stability-levels) documentation.
```

Reply:
Yes it is required for using `pyroscope.scrape`: https://github.com/grafana/alloy/blob/579fd4faa684c7ca3be8ecfad95ff0fd297a3fbe/internal/component/pyroscope/scrape/scrape.go#L39-L40

Reply:
I updated the suggestion with your explanation. Thank you.